### PR TITLE
Feat/sync sensitive dirs flag

### PR DESCRIPTION
**Pull request by zhuk-aa**
**Context** 
To be able to run `honest_benchmark` nightly with github workflows. Atm when I'm trying to run it - cli expects me to manually select between yes or no

**What's implemented in this PR?**
1. renamed `ignoreUnsafe` -> `syncSensitive`
2. reading it in `index.ts`

### CODE REVIEW

File: .github/workflows/release-to-discord.yml [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   github-releases-to-discord:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        
       - name: Github Releases To Discord
-        uses: sillyangel/releases-to-discord@v1.0.0
+        uses: sillyangel/releases-to-discord@v1
```
Comment (BonnardValentin) [inline .github/workflows/release-to-discord.yml:14]: https://github.com/sillyangel/releases-to-discord/releases/tag/v1  , you should keep v1

**Review by BonnardValentin — APPROVED** [review]

File: .github/workflows/release-to-discord.yml (hunk 1 above)
Comment (BonnardValentin) [reply .github/workflows/release-to-discord.yml:14]: it's a tag , not a version  so it's v1

Comment (zhuk-aa) [reply .github/workflows/release-to-discord.yml:14]: ah damn, that's some branching issue.
Didn't intent to make this change

**Comment by zhuk-aa** [comment]
Less important now - able to run on CI"CD without this change. So let's merge it only if we see a real need

**Review by BonnardValentin — APPROVED** [review]

**Comment by BonnardValentin** [comment]
> Less important now - able to run on CI"CD without this change. So let's merge it only if we see a real need

Do you talk about that `sillyangel/releases-to-discord@v1` ?  I'm afraid that without v1 instead of v1.0.0 -> don't work

**Comment by zhuk-aa** [comment]
Sry, wasn't clear. Meant this feature overall
> Do you talk about that sillyangel/releases-to-discord@v1 ? I'm afraid that without v1 -> don't work

**Comment by BonnardValentin** [comment]
Just a thought but  if we need these changes to be able to run this CI/CD pipeline, maybe we should consider: What can we change in honest_benchmark so that it is capable of not creating/using sensitive directories ?

**Pull request closed without merging**